README.md

```diff
@@ -2,18 +2,107 @@
 
 ## Overview
 
-Think of the interactions you have with any web application. You're likely clicking buttons, hovering over items, submitting forms - these are just a few of the actions you can take to interact with the application. If you've ever tried to manage all of these actions with plain old JavaScript of jQuery, you know how hard it can be to keep track of all of those different actions. In this reading, we'll look at how the Flux architecture helps us describe these actions in a large application. 
+Think of the interactions you have with any web application. You're likely clicking buttons, hovering over items, submitting forms - these are just a few of the actions you can take to interact with the application. If you've ever tried to manage all of these actions with plain old JavaScript or jQuery, you know how hard it can be to keep track of all of those different actions. In this reading, we'll look at how Redux helps us describe these actions in a large application. 
 
 ## Objectives
 1. Understand the purpose of actions in a Redux application
 2. Build an action creator function
 3. Use an action creator with data
 
-## Outline
+## Life Before Actions
 
-1. Describe the concept of actions in general
-2. Explain that actions are Plain Old JavaScript Objects an action type
-3. Setup a ToDo List Application and pass an `ADD_TODO` action to the store
-4. Explain the need for an action creator in case any action needs to change
-5. Mention that action creators are just Plain Old JavaScript Functions that return objects. 
-6. Show how the action gets passed into dispatch
+As we've seen in the past, keeping track of user interactions is hard. Figuring out when and where we're firing web requests and updating our views can get confusing very quickly. Imagine you're making a twitter clone using only vanilla JavaScript. It's really confusing.
+
+## Time for Some Actions
+
+In Redux, we have a simple way to describe our actions. We simply create plain old JavaScript objects with a `type` property. The type property may be something like `ADD_TODO`, `CREATE_TWEET`, or `INCREMENT_COUNT`. 
+
+```javascript
+let incrementCountAction = {type: 'INCREMENT_COUNT'}
+let addTodoAction = {type: 'ADD_TODO'}
+let createTweetAction = {type: 'CREATE_TWEET'}
+```
+
+Imagine that we have a `store` object who's job it is to keep track of different pieces of data for our application. We'll refer to those pieces of data as our `state`. 
+
+```javascript
+store.getState() // returns the current state
+```
+
+We can pass actions to our store using a method called dispatch. We'll then depend on our store to update the state of our application and update any views. 
```

Comment:
Because the students have yet to see an example of the store object and how it stores state (and uses dispatch) do we want to show a brief snippet of what our store object looks like? Either that or a reassurance that they will cover the store in depth moving forward